docs/rsdl/rapid-rsdl-abnf.md

```diff
@@ -47,27 +47,20 @@ structuredType       = annotations [ %s"abstract" RWS ] %s"type" RWS identifier
 
 structuredTypeMember = property / operation ; property, action, or function
 
-property             = structuralProperty / referenceProperty
+property             = singlePropertyDefinition [ OWS (primitivePropertyCapabilities / singleNavigationCapabilities) ]
+                     / collectionPropertyDefinition [ OWS ( collectionCapabilities / collectionNavigationCapabilities) ]
 
-structuralProperty   = primitiveProperty / collectionProperty
+singlePropertyDefinition  = annotations [propertyModifier RWS] identifier OWS ":" OWS singleTypeReference
 
-primitiveProperty    = propertyDefinition [ primitivePropertyCapabilities ]
+collectionPropertyDefinition  = annotations [propertyModifier RWS] identifier OWS ":" OWS collectionTypeReference
```

Comment:
If collection properties can't be key, we could remove the `[propertyModifier RWS]` part

Reply:
good catch!